Lib/pdb.py

```diff
@@ -588,6 +588,28 @@ def _hold_exceptions(self, exceptions):
             self._chained_exceptions = tuple()
             self._chained_exception_index = 0
 
+    def process_cmdqueue(self):
+        """Process commands in the command queue.
+
+        The function returns in two cases:
+            1. the command queue is empty
+            2. a command to continue execution is encountered
+
+        The return value is whether a following cmdloop should be executed, or
```

Comment:
This sentence seems truncated.

Reply:
I intended to say "or True for the first case and False for the second", maybe that's not the best expression?